index.html

```diff
@@ -114,159 +92,132 @@ <h2>Custom software built for your everyday business needs.</h2>
         <div class="row d-flex align-items-center">
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-1.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-2.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-3.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-4.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-5.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
           <div class="col-lg-2 col-md-4 col-6">
-            <img src="assets/img/clients/client-6.png" class="img-fluid" alt="">
+            <img src="#" class="img-fluid" alt="">
           </div>
         </div>
       </div>
-    
-    </section>
-  -->
-    <!-- End Clients Section -->
-
+    </section>-->
+          <!-- End Clients Section -->
    <!-- ======= Services Section ======= -->
    <section id="services" class="services">
     <div class="container">
-
       <div class="section-title">
         <span>Services</span>
         <h2>Services</h2>
       </div>
-
       <div class="row">
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch" data-aos="fade-up">
           <div class="icon-box">
-            <div class="icon"><i class="bx bxl-dribbble"></i></div>
+            <div class="icon"><i class="bi-code-square"></i></div>
             <h4><a href="">Application Development & Mobile Application Development</a></h4>
             <p>We have a strong understanding of programming languages and the ability to solve complex problems. We also are able to work well in a team and communicate effectively with others. Additionally, we have deep understanding of the latest technologies and trends in the field</p>
           </div>
         </div>
-
         <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="fade-up" data-aos-delay="150">
           <div class="icon-box">
-            <div class="icon"><i class="bx bx-file"></i></div>
+            <div class="icon"><i class="bi-file-earmark-binary"></i></div>
             <h4><a href="">System Automation</a></h4>
             <p>Authorative use of software, hardware and other tools to automate the administration, management, monitoring, and other operational tasks of computing. We have a strong understanding of a variety of programming languages and experience with automation tools and frameworks.</p>

```

Comment:
```suggestion
            <p> Authoritative use of software, hardware and other tools to automate the administration, management, monitoring, and other operational tasks of computing. We have a strong understanding of various programming languages and experience with automation tools and frameworks.</p>

```
Authoritative isn't spelled right.

Reply:
lol also if you want you can ignore the change from "a variety of " to "various" it makes no difference!